<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #deede3;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    background-color: var(--jade-50);
}

QLabel {
    color: var(--jade-500);
    font-size: 16px;
    font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>Presentaciones</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>691</width>
      <height>391</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ver, agregar o editar presentacion&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget {
    background-color: #deede3;
    border: 1px solid #cccccc;
    border-radius: 4px;
}

QTabWidget::tab-bar {
    left: 5px;
}

QTabWidget::tab {
    background-color: #deede3;
    border: 1px solid #1e362d;
    border-bottom-left-radius: 4px;
    border-bottom-right-radius: 4px;
    padding: 6px 12px;
}

QTabWidget::tab:selected {
    background-color: #c0dacb;
    border: 1px solid #1e362d;
    border-bottom: 1px solid #c0dacb;
}

QTabWidget::pane {
    background-color: #deede3;
    border: 1px solid #1e362d;
    border-top: none;
    border-bottom-left-radius: 4px;
    border-bottom-right-radius: 4px;
}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Listado</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>641</width>
        <height>321</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    border: 2px solid #1e362d;
    border-radius: 8px;
    padding: 10px;
    background-color: var(--jade-50);
}

QGroupBox:title {
    color: #1e362d;
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding-left: 5px;
    padding-right: 5px;
    border: none;
}</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>73</width>
         <height>27</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget {
    background-color: var(--jade-50);
}

QLabel {
    color: var(--jade-500);
    font-size: 16px;
    font-weight: bold;
}</string>
       </property>
       <property name="text">
        <string>Nombre</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtBuscar">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>120</width>
         <height>35</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Inserte el dato para buscar&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #96bfab;
    border: 1px solid #cccccc;
    padding: 6px;
    border-radius: 4px;
    font-size: 14px;
}

QLineEdit:hover {
    border: 1px solid #689f84;
}

QLineEdit:focus {
    border: 1px solid #689f84;
    outline: none;
}

QLineEdit::placeholder {
    color: #1e362d;
    font-weight: bold;
}</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnBuscar">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>20</y>
         <width>85</width>
         <height>37</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #c0dacb, stop: 1 #96bfab);
    color: #ffffff;
    border: 1px solid #1e362d;
    padding: 8px 16px;
    border-radius: 4px;
    font-size: 14px;
}

QPushButton:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #689f84, stop: 1 #96bfaa);
    border: 1px solid #96bfaa;
}

QPushButton:pressed {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #2a5242, stop: 1 #32624d);
    border: 1px solid #32624d;
}</string>
       </property>
       <property name="text">
        <string>Buscar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnEliminar">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>20</y>
         <width>85</width>
         <height>37</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #c0dacb, stop: 1 #96bfab);
    color: #ffffff;
    border: 1px solid #1e362d;
    padding: 8px 16px;
    border-radius: 4px;
    font-size: 14px;
}

QPushButton:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #689f84, stop: 1 #96bfaa);
    border: 1px solid #96bfaa;
}

QPushButton:pressed {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #2a5242, stop: 1 #32624d);
    border: 1px solid #32624d;
}</string>
       </property>
       <property name="text">
        <string>Eliminar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnImprimir">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>20</y>
         <width>85</width>
         <height>37</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #c0dacb, stop: 1 #96bfab);
    color: #ffffff;
    border: 1px solid #1e362d;
    padding: 8px 16px;
    border-radius: 4px;
    font-size: 14px;
}

QPushButton:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #689f84, stop: 1 #96bfaa);
    border: 1px solid #96bfaa;
}

QPushButton:pressed {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #2a5242, stop: 1 #32624d);
    border: 1px solid #32624d;
}</string>
       </property>
       <property name="text">
        <string>Imprimir</string>
       </property>
      </widget>
      <widget class="QTableView" name="tbDatos">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>531</width>
         <height>211</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QTableView {
    background-color: #f2f7f4;
    border: 1px solid #cccccc;
    selection-background-color: #96bfaa;
    selection-color: #ffffff;
}

QTableView QHeaderView {
    background-color: #c0dacb;
    color: #333333;
    border: none;
}

QTableView QHeaderView::section {
    background-color: #deede3;
    color: #333333;
    padding: 6px;
    border: 1px solid #1e362d;
}

QTableView QHeaderView::section:checked {
    background-color: #96bfaa;
    color: #ffffff;
}

QTableView::item {
    padding: 6px;
}

QTableView::item:selected {
    background-color: #96bfaa;
    color: #ffffff;
}

QTableView::item:focus {
    background-color: #96bfaa;
    color: #ffffff;
    outline: none;
}</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Mantenimiento</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>451</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    border: 2px solid #1e362d;
    border-radius: 8px;
    padding: 10px;
    background-color: var(--jade-50);
}

QGroupBox:title {
    color: #1e362d;
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding-left: 5px;
    padding-right: 5px;
    border: none;
}</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>55</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget {
    background-color: var(--jade-50);
}

QLabel {
    color: var(--jade-500);
    font-size: 16px;
    font-weight: bold;
}</string>
       </property>
       <property name="text">
        <string>Codigo</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>63</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget {
    background-color: var(--jade-50);
}

QLabel {
    color: var(--jade-500);
    font-size: 16px;
    font-weight: bold;
}</string>
       </property>
       <property name="text">
        <string>Nombre</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget {
    background-color: var(--jade-50);
}

QLabel {
    color: var(--jade-500);
    font-size: 16px;
    font-weight: bold;
}</string>
       </property>
       <property name="text">
        <string>Descripcion</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnEditar">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>180</y>
         <width>85</width>
         <height>37</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #c0dacb, stop: 1 #96bfab);
    color: #ffffff;
    border: 1px solid #1e362d;
    padding: 8px 16px;
    border-radius: 4px;
    font-size: 14px;
}

QPushButton:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #689f84, stop: 1 #96bfaa);
    border: 1px solid #96bfaa;
}

QPushButton:pressed {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #2a5242, stop: 1 #32624d);
    border: 1px solid #32624d;
}</string>
       </property>
       <property name="text">
        <string>Editar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnGuardar">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>180</y>
         <width>85</width>
         <height>37</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #c0dacb, stop: 1 #96bfab);
    color: #ffffff;
    border: 1px solid #1e362d;
    padding: 8px 16px;
    border-radius: 4px;
    font-size: 14px;
}

QPushButton:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #689f84, stop: 1 #96bfaa);
    border: 1px solid #96bfaa;
}

QPushButton:pressed {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #2a5242, stop: 1 #32624d);
    border: 1px solid #32624d;
}</string>
       </property>
       <property name="text">
        <string> Guardar </string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnSalir">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>180</y>
         <width>85</width>
         <height>37</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #c0dacb, stop: 1 #96bfab);
    color: #ffffff;
    border: 1px solid #1e362d;
    padding: 8px 16px;
    border-radius: 4px;
    font-size: 14px;
}

QPushButton:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #689f84, stop: 1 #96bfaa);
    border: 1px solid #96bfaa;
}

QPushButton:pressed {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #2a5242, stop: 1 #32624d);
    border: 1px solid #32624d;
}</string>
       </property>
       <property name="text">
        <string>Salir</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtCodigo">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>120</width>
         <height>35</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #96bfab;
    border: 1px solid #cccccc;
    padding: 6px;
    border-radius: 4px;
    font-size: 14px;
}

QLineEdit:hover {
    border: 1px solid #689f84;
}

QLineEdit:focus {
    border: 1px solid #689f84;
    outline: none;
}

QLineEdit::placeholder {
    color: #1e362d;
    font-weight: bold;
}</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtNombre">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>70</y>
         <width>120</width>
         <height>35</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #96bfab;
    border: 1px solid #cccccc;
    padding: 6px;
    border-radius: 4px;
    font-size: 14px;
}

QLineEdit:hover {
    border: 1px solid #689f84;
}

QLineEdit:focus {
    border: 1px solid #689f84;
    outline: none;
}

QLineEdit::placeholder {
    color: #1e362d;
    font-weight: bold;
}</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="txtDescripcion">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>120</y>
         <width>221</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QTextEdit {
    background-color: #96bfab;
    border: 1px solid #cccccc;
    padding: 6px;
    border-radius: 4px;
    font-size: 14px;
}

QTextEdit:hover {
    border: 1px solid #689f84;
}

QTextEdit:focus {
    border: 1px solid #689f84;
    outline: none;
}</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
